package.json

```diff
@@ -5,31 +5,41 @@
   "author": "Adam Klein <adam@500tech.com>",
   "homepage": "https://github.com/500tech/angular2-tree-component",
   "license": "MIT",
-  "main": "lib/angular2-tree",
-  "typings": "lib/defs/angular2-tree.d.ts",
+  "main": "angular2-tree-component",
+  "typings": "angular2-tree-component.d.ts",
   "keywords": ["angular2", "tree", "angular2-tree", "ng2", "ng2-tree"],
   "scripts": {
-    "test": "karma start"
+    "test": "karma start",
+    "clean": "rimraf node_modules && npm run clean:typescript && npm cache clean",
+    "clean:typescript": "rimraf angular2-tree.js angular2-tree.d.ts ./lib/**/*.js ./lib/components/*.d.ts ./lib/constants/*.d.ts ./lib/defs/api.d.ts ./lib/models/*.d.ts",
+    "build": "tsc",
+    "postinstall": "typings install"
   },
   "contributors": [
     "Adam Klein <adam@500tech.com>"
   ],
-  "peerDependencies": {
-    "@angular/common":  "2.0.0-rc.1",
-    "@angular/compiler":  "2.0.0-rc.1",
-    "@angular/core":  "2.0.0-rc.1",
-    "@angular/http":  "2.0.0-rc.1",
-    "@angular/platform-browser":  "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic":  "2.0.0-rc.1",
-    "@angular/router":  "2.0.0-rc.1",
-    "@angular/router-deprecated":  "2.0.0-rc.1",
+  "dependencies": {
```

Comment:
As I understand, peerDependencies is better for angular2, because you don't want to bring multiple versions of Angular into the system, just use the one the user has provided in his/her package.json.

Is there an advantage in using actual dependencies?